quic/s2n-quic-transport/src/stream/stream_container.rs

```diff
@@ -206,7 +264,7 @@ impl<S: StreamTrait> InterestLists<S> {
             waiting_for_frame_delivery_link,
             waiting_for_frame_delivery
         );
-        sync_interests!(
+        sync_trans_interests!(
```

Comment:
I'm wondering if it would be more intuitive to have the batching logic in `stream_container::iterate_transmission_list` rather in `update_interests`